torchvision/datapoints/_datapoint.py

```diff
@@ -25,9 +32,13 @@ def _to_tensor(
             requires_grad = data.requires_grad if isinstance(data, torch.Tensor) else False
         return torch.as_tensor(data, dtype=dtype, device=device).requires_grad_(requires_grad)
 
+    @classmethod
+    def _wrap(cls: Type[D], tensor: torch.Tensor) -> D:
+        return tensor.as_subclass(cls)
```

Comment:
I ended up moving `_wrap` and `wrap_like` in the base class. They were all implemented exactly in the same way for all datapoints except for `BoundingBoxes`, because that one requires meta-data.

I think it's OK to assume that these are a good default implementations for most datapoints, i.e. those that don't require meta-data. What can go wrong anyway?

It makes the example much easier to write and explain, so I think we should do that. I can do it in another PR if you prefer though.

Reply:
I'm ok leaving it here.